src/pretix/base/forms/questions.py

```diff
@@ -630,6 +630,10 @@ def __init__(self, *args, **kwargs):
         )
         self.fields['state'].widget.is_required = True
 
+        if cc and cc not in EU_COUNTRIES and fprefix + 'vat_id' in self.data:
+            self.data = self.data.copy()
+            del self.data[fprefix + 'vat_id']
```

Comment:
Modifying ``self.data`` doesn't feel clean, I think the part in the ``clean()`` should suffice?

Reply:
Not important, but the idea is to remove the input if JavaScript is disabled and the form is reinitialized with a selecte country that is outside the EU, similar to https://github.com/pretix/pretix/commit/b0b0f7474d7999a088444e0f1c0649f18397585d
![grafik](https://user-images.githubusercontent.com/44349054/86355292-c4a62700-bc6a-11ea-9d1b-ad845e605b00.png).

Reply:
Ok, got it. Maybe add a small comment for future readers? :)

Reply:
Thanks, I added a comment.